src/js/trove/particle.js

```diff
@@ -70,12 +67,38 @@ define(["js/runtime-util", "js/ffi-helpers", "trove/json", "trove/world", "trove
 
 
           //////////////////////////////////////////////////////////////////////
-          
-          var ToParticle = function(handler, ename, sdict) {
+
+          var sendEvent = function(ename, data, options) {
+            var xhr = new XMLHttpRequest();
```

Comment:
Meta-comment: I wonder if we should just allow ourselves a hard dependency on JQuery or not.  XMLHttpRequest is kind of a pain IMO – how annoying do you find raw XMLHttpRequest?